packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,32 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/**
+ * FrameContexts provides information about the context in which the app is running within the host.
+ * Developers can use FrameContexts to determine how their app should behave in different contexts,
+ * and can use the information provided by the context to adapt the app to the user's needs.
+ * 
+ * @example 
+ * If the app is running in the settings context, the developer may want to display options for the 
+ * user to configure their preferences. If the app is running in the content context, the developer 
```

Comment:
I think the example would be clearer if it was more definitive. If one is running in the `settings` context, it's not that your app _may_ want to display options -- that's literally what that context is for. I don't know a ton about this but based on the link I put the next comment maybe consider something like this?

```
If your app is running in the `settings` context, you should be displaying your apps configuration page.
```

Reply:
Will update in latest commit.